docs/content/patterns/crud-data-tables/index.mdx

```diff
@@ -0,0 +1,448 @@
+---
+title: Create, read, update and delete data in tables
+description: Applications that require persistent data storage commonly require table layouts to present stored data. Here are some patterns that enable users to create, read, update and delete persistent data in tables.
+relatedComponents:
+  ['button', 'file-input', 'file-upload', 'section-alert', 'table']
+---
+
+## Overview
+
+Use CRUD table interactions to help users to Create, Read, Update or Delete (CRUD) data presented in tables. These common interactions can be achieved by providing an additional drawer or subpage surface for the action, to reduce the length and complexity of the form page and to provide the user content that is relevant to the interaction through progressive disclosure.
+
+### Create a record in a data table
+
+When creating a record to a data table is the primary action of the page, provide a primary button immediately before the table so it is above the table in the reading order and page hierarchy.
```

Comment:
```suggestion
When adding a record to a data table is the primary action of the page, provide a primary button immediately before the table so it is above the table in the reading order and page hierarchy.
```

Reply:
I'd also say "so it is above the table in the reading order and page hierarchy" is unnecessary

Reply:
I have intentionally changed the language to align with the acronym CRUD C = create. Please maintain the CRUD acronym language throughout

Reply:
I thought that, but the one below had "adding" instead if creating. "Creating a record to a table" reads really strangely though.

Reply:
we need to reference the position on the page if you dont think the hierarchy is required

Reply:
The hierarchy and placement is good, it's just the start to the paragraph, "When creating a record to a data table". You can't create to something